### PR TITLE
Add ECSI.net

### PR DESCRIPTION
**Pull request by gopherit**

### CODE REVIEW

**Comment by J0WI** [comment]
It seems that they have moved to `heartlandecsi.com`

**Comment by gopherit** [comment]
Only ecsi.net and www.ecsi.net redirect o www.heartlandecsi.com, and it's been that way for years.  The rest of the ecsi.net subdomains are correct in the ruleset, and many of them aren't automatically redirecting to https, so the rules help with that.

File: src/chrome/content/rules/ECSI.net.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	appmail (Invalid Certificate)
+	corp (Redirects to www.heartlandecsi.com)
+	vpn (Self-signed)
+-->
+
+<ruleset name="ECSI.net">
+	<target host="www.ecsi.net" />
+	<target host="borrower.ecsi.net" />
+	<target host="cgi.ecsi.net" />
+	<target host="client.ecsi.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/ECSI.net.xml]: easypath.ecsi.net

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**